src/PrivateDns/PrivateDns/help/New-AzPrivateDnsVirtualNetworkLink.md

```diff
@@ -0,0 +1,235 @@
+
+## EXAMPLES
+
+### Example 1: Create a Private DNS virtual network link
+```
+PS C:\>$Link = New-AzPrivateDnsVirtualNetworkLink -ZoneName "myzone.com" -ResourceGroupName "MyResourceGroup" -Name "mylink" -VirtualNetworkId "myvirtualnetwork" -EnableRegistration
+
+Name                    : mylink
+ResourceId              : /subscriptions/xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/MyResourceGroup/providers/Microsoft.N
+                          etwork/privateDnsZones/myzone.com/virtualNetworkLinks/mylink
+ResourceGroupName       : MyResourceGroup
+ZoneName                : myzone.com
+VirtualNetworkId        : /subscriptions/xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/MyResourceGroup/providers/Microsoft.N
+                          etwork/virtualNetworks/myvirtualnetwork
+Location                :
+Etag                    : "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+Tags                    : {}
+RegistrationEnabled     : True
+VirtualNetworkLinkState : Completed
+ProvisioningState       : Succeeded
+```
+
+This command creates a new virtual network link associated with the private DNS zone named myzone.com and virtual network "myvirtualnetwork" (which has already been created in the resource group) in the specified resource group, and then stores it in the $Link variable.
+
+## PARAMETERS
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with azure
+
+```yaml
+Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzContext, AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -EnableRegistration
+Switch parameter that represents if the link is registration enabled or not.
+
+```yaml
+Type: System.Management.Automation.SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Name
+Specifies the name of the virtual network link to create.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ResourceGroupName
+Specifies the resource group in which to create the link.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Tag
+A hash table which represents resource tags.
+
+```yaml
+Type: System.Collections.Hashtable
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -VirtualNetwork
+The resource id of the virtual network associated with the link.
```

Comment:
Fix this

Reply:
This was not fixed - this parameter is not an id, it's an object.

Reply:
not sure why this change was not reflected in the last iteration, fixed it now.